server/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node index.js",
+    "start": "node ./test/index.test.js --recursive",
```

Comment:
Why does our start script appear to utilize a test file?

Reply:
Big mistake lol, will fix